File: src/fib.kt
fun fib(n: Int): Int {
  if (n < 3) return 1;
  return fib(n - 1) + fib(n - 2)
}

fun main(args: Array<String>) {
  println(fib(40));
}

// kotlinc fib.kt -include-runtime -d fib.jar
// time java -jar fib.jar

File: src/fib.cs
using System;

namespace feibi
{
    class Program
    {
        static int fib( int n)
        {
            if (n < 3)
                return 1;
            else
                return fib(n - 2) + fib(n - 1);
        }

        static void Main(string[] args)
        {
            Console.WriteLine("fib(40) = {0}", fib(40));
        }
    }
}

// $ mono-csc fib.cs
// $ time ./fib.exe
